tests/phpunit/tests/blocks/register.php

```diff
@@ -515,6 +515,14 @@ public function test_block_registers_with_metadata_fixture() {
 			$result->provides_context
 		);
 		$this->assertSameSets( array( 'groupId' ), $result->uses_context );
+		$this->assertSame(
+			array( 'root' => '.editor-only.wp-block-notice' ),
+			$result->editor_selectors
+		);
+		$this->assertSame(
+			array( 'root' => '.wp-block-notice' ),
+			$result->selectors
+		);
```

Comment:
Please add assertion messages to each of these `assertSame()`. Why? If any fail, the message will identify which of the assertions failed.

Reply:
Also add `@ticket 57585` to the test method.